demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1213,26 +1220,6 @@ def get_packs(changed_files):
 
         return packs
 
-    @staticmethod
-    def get_id_set_file(skip_id_set_creation, id_set_path):
```

Comment:
Why are we removing this? we need a new/valid id_set file for the id set checks and dependencies.

Reply:
We are creating it if we pass the correct flag. so this funcis redundant.
```
# id_set validation
        if self.id_set_validations:
            id_set_validator = IDSetValidator(is_circle=self.is_circle, configuration=Configuration(),
                                              ignored_errors=pack_error_ignore_list,
                                              print_as_warnings=self.print_ignored_errors)
            if not id_set_validator.is_file_valid_in_set(file_path, file_type):
                return False
```